test/src/main/java/org/jvnet/hudson/test/recipes/WithPlugin.java

```diff
@@ -50,9 +50,9 @@
 @Retention(RUNTIME)
 public @interface WithPlugin {
     /**
-     * Name of the plugin.
+     * Whitespace separated list of plugin names.
```

Comment:
OK, I changed the regex to parse a CSV value.

I thought about using an array. But for compatibility, the non-array value must be kept. And if there are both, array and non-array, which should take precedence? Isn't it confusing to have both?